homeworks/serhii.noskov_SerhiiNSN/5-logic/homework.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * @param firstValue string with first number
+ * @param secondValue string with second number
+ * @param operation string with operation symbol
+ * @returns {string|number} result of calculation or error message:
+ * - if firstValue or secondValue is not a number, return 'Enter a number'
+ * - if operation is not '+', '-', '*', '/', return 'Choose a valid operation'
+ * - if the result is more than 100, return 'Result is too big'
+ * - else return number of result: 1 '+' 2 returns 3
+ *
+ */
+
+function calculate(firstValue, secondValue, operation) {
+    const num1 = Number(firstValue);
+    const num2 = Number(secondValue);
+    const addition = num1 + num2;
```

Comment:
Looks like you calculate all 4 results for every case.
Maybe you can calculate each result only for its specific case?
Now these operations are simple, but in real projects, this could be a reason for performance issue.